presto-main/src/main/java/com/facebook/presto/geospatial/BingTileFunctions.java

```diff
@@ -547,29 +546,6 @@ private static double addDistanceToLatitude(
         return newLatitude;
     }
 
-    private static BingTile[] getTilesInBetween(BingTile leftUpperTile, BingTile rightLowerTile, int zoomLevel)
-    {
-        checkArgument(leftUpperTile.getZoomLevel() == rightLowerTile.getZoomLevel());
-        checkArgument(leftUpperTile.getZoomLevel() > zoomLevel);
-
-        int divisor = 1 << (leftUpperTile.getZoomLevel() - zoomLevel);
-        int minX = (int) Math.floor(leftUpperTile.getX() / divisor);
-        int maxX = (int) Math.floor(rightLowerTile.getX() / divisor);
-        int minY = (int) Math.floor(leftUpperTile.getY() / divisor);
-        int maxY = (int) Math.floor(rightLowerTile.getY() / divisor);
-
-        BingTile[] tiles = new BingTile[(maxX - minX + 1) * (maxY - minY + 1)];
-        int index = 0;
-        for (int x = minX; x <= maxX; x++) {
-            for (int y = minY; y <= maxY; y++) {
-                tiles[index] = BingTile.fromCoordinates(x, y, OPTIMIZED_TILING_MIN_ZOOM_LEVEL);
-                index++;
-            }
-        }
-
-        return tiles;
-    }
-
```

Comment:
the method below it is also unused. Can you remove it too?

Reply:
done